R/eml.R

```diff
@@ -73,11 +73,11 @@ pid_to_eml_entity <- function(mn,
 
 #' Create EML physical objects for the given set of PIDs
 #'
-#' This is a wrapper around [sysmeta_to_eml_physical()] which handles the task of
-#' creating the EML physical.
+#' This function creates a data object's physical.
 #'
 #' @param mn (MNode) Member Node where the PID is associated with an object.
 #' @param pid (character) The PID of the object to create the physical for.
+#' @param headers (double) The number of headers in a csv file. Default is equal to 1.
```

Comment:
I'd suggest `num_header_lines` as the argument here maybe and specify that it is the number of header lines